ui/app/templates/components/clients/monthly-usage.hbs

```diff
@@ -14,38 +13,29 @@
   <div class="chart-subTitle">
     <h2 class="chart-title">Total monthly clients</h2>
     <p class="chart-subtext">
-      Each unique client is counted once per month. This can help with capacity planning.
+      Each client is counted once per month. This can help with capacity planning.
     </p>
   </div>
 
   <div class="data-details-top">
     <h3 class="data-details">Average total clients per month</h3>
     <p class="data-details">
-      {{#if this.averageTotalClients}}
-        {{format-number this.averageTotalClients}}
-      {{else}}
-        <span class="has-text-danger is-size-8">
```

Comment:
Removed because if there is an error collecting data for this component - then it shouldn't render at all